examples/fast-api-server/fast_api_server/main.py

```diff
@@ -11,7 +11,11 @@
 # create a library with the custom function
 custom_python_lib = PythonCRUD(functions=[find_email_by_name_function])
 
-hooli_assistant = Assistant(libraries=["piedpiper", custom_python_lib])
+hooli_assistant = Assistant(
+    libraries=["piedpiper", custom_python_lib],
+    scope_description="""Only answer questions about Hooli company related matters.
+Also answer question that refers anything in the current chat history.""",
```

Comment:
`Also answer questions that refer to anything in the chat history` potentially?

Reply:
A prompt engineering only strategy is a good first step, but if this needs to be strongly enforced then additional safeguards are necessay.